pkg/protocol/engine/mempool/spenddag/events.go

```diff
@@ -0,0 +1,47 @@
+package spenddag
+
+import (
+	"github.com/iotaledger/hive.go/ds"
+	"github.com/iotaledger/hive.go/runtime/event"
+)
+
+// region Events ///////////////////////////////////////////////////////////////////////////////////////////////////////
+
+// Events is a container that acts as a dictionary for the events of a SpendDAG.
+type Events[SpendID, ResourceID comparable] struct {
+	// SpendCreated is triggered when a new Conflict is created.
+	SpendCreated *event.Event1[SpendID]
+
+	// SpendEvicted is triggered when a Conflict is evicted from the SpendDAG.
+	SpendEvicted *event.Event1[SpendID]
+
+	// ConflictingResourcesAdded is triggered when the Conflict is added to a new ConflictSet.
```

Comment:
```suggestion
	// ConflictingResourcesAdded is triggered when the Spend is added to a new ConflictSet.
```

Reply:
Is that still called `ConflictSet`?

Reply:
Yes, a set of conflicting spends is still called ConflictSet.